dhall-json/src/Dhall/JSONToDhall.hs

```diff
@@ -590,6 +590,77 @@ dhallFromJSON (Conversion {..}) expressionType =
 
           return result
 
+    -- Arbitrary JSON ~> https://prelude.dhall-lang.org/JSON/Type (v13.0.0 <=)
```

Comment:
Nice! I hadn't realized that `json-to-dhall` supports this at all!

Reply:
Yeah, this is documented here: https://github.com/Gabriel439/dhall-manual/blob/a0b4ceb9fead3ba151fc28f3b389573493b66f6b/manuscript/01-YamlToDhall.md